specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/dscConfiguration.json

```diff
@@ -0,0 +1,667 @@
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "configurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The create or update parameters for configuration."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/DscConfigurationUpdateParameters"
+            },
+            "description": "The create or update parameters for configuration."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscConfiguration"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/configurations/{configurationName}/content": {
+      "get": {
+        "tags": [
+          "DscConfiguration"
+        ],
+        "operationId": "DscConfiguration_GetContent",
+        "description": "Retrieve the configuration script identified by configuration name.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/configurationoperations"
+        },
+        "x-ms-examples": {
+          "Get DSC Configuration content": {
+            "$ref": "./examples/getDscConfigurationContent.json"
+          }
+        },
+        "produces": [
+          "text/powershell"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "configurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The configuration name."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "string"
```

Comment:
Need to change this to file type. - https://github.com/Azure/azure-rest-api-specs/pull/3749/files